packages/storybook/src/nimble/select/select.stories.ts

```diff
@@ -240,6 +259,14 @@ const metadata: Meta<SelectArgs> = {
             description: 'Emitted when the user types in the filter input.',
             table: { category: apiCategory.events },
             control: false
+        },
+        width: {
+            name: 'width',
+            description:
```

Comment:
I confirmed that width is working as I'd expect now. If it's set to `auto` or unset then setting a long label or picking a long option will grow the width. If it's set to a fixed length like a `px` or `%` value then long labels will wrap and long options will ellipsize. 👍

Reply:
I'm not really seeing the need to document this explicitly. Our controls behave like standard html elements. Setting the width sets the width. Why would we call it out only for this specific control?

If we have sizing style usage guidance we should document that in the mdx but the current description isn't really usage guidance, just a general description of how CSS width properties work.

Reply:
Fair point - @jattasNI - thoughts on the limiting principle for this type of documentation?

The latest push removes the width args…

Reply:
I am ok with removing the arg and agree that the description was pretty general.

Longer term I think sizing usage guidance under Styling>>Sizing would be useful for all components. If we wanted to try to start doing that here I think it could look something like this:
> This component's height is fixed and its width obeys standard CSS sizing rules. If the component has no explicit `width` (e.g. `width: auto`) then it will grow to fit both the label and the currently selected item. When it reaches its `max-width` the label will wrap and the selected item will ellipsize. Applications with fixed layouts that shouldn't change in response to user interaction should set an explicit width.

@fredvisser your call whether to include something like this now or save it for a more dedicated effort across all components. And if we are including it, @rajsite you're welcome to wordsmith with more precise CSS terminology.

Reply:
That is a good textual representation of what the control does in text form. As a user that tells me the control behavior I see is documented. That could be useful but seems like a lot of work to document and maybe is redundant? I think if someone really wanted to know those behaviors it should be covered in a matrix test and can see what long labels look like there (i.e. if their goal is validating what they see is expected).

If a user is coming to the style section to know how they should style a control I don't think all the detail helps. I just want to know what to do as quickly as possible so someone from Nimble or UX won't complain about how I sized the control.

I'd say simplify to something that gives both reasoning and guidance to build decisions on:
- By default the select control resizes to the selected option size which keeps the content visible and the control compact.
- In a grid layout consider giving the select a specific width to align the size with other grid items. The control should be wide enough so labels do not wrap.

Reply:
Added the styling guidance as…

>## Styling

>By default, the select control resizes to the selected option size. In most
>layouts, consider giving the select a specific `width` to align with other
>items. Ensure the control is wide enough so labels do not wrap.